apps/ledger-live-desktop/tests/specs/speculos/erc20.spec.ts

```diff
@@ -2,24 +2,28 @@ import test from "../../fixtures/common";
 import { specs } from "../../utils/speculos";
 import { Application } from "tests/page";
 import { addTmsLink } from "tests/fixtures/common";
+import { Token } from "../../enum/Tokens";
 
-test.use({
-  userdata: "speculos-tests-app",
-  testName: "tokenERC20",
-  speculosCurrency: specs["Ethereum".replace(/ /g, "_")],
-  speculosOffset: 0,
-});
-const token = "Tether USD";
+const tokens: Token[] = [Token.ETH_USDT];
 
-test.describe.parallel("ERC20 token", () => {
-  test(`Check ERC20 token`, async ({ page }) => {
-    addTmsLink(["B2CQA-1079"]);
+for (const [i, token] of tokens.entries()) {
+  test.describe.parallel("ERC20 token", () => {
+    test.use({
+      userdata: "speculos-tests-app",
+      testName: "tokenERC20",
+      speculosCurrency: specs[token.parentAccount.currency.deviceLabel.replace(/ /g, "_")],
+      speculosOffset: i,
+    });
 
-    const app = new Application(page);
+    test(`Check ERC20 token`, async ({ page }) => {
+      addTmsLink(["B2CQA-1079"]);
 
-    await app.layout.goToPortfolio();
-    await app.portfolio.navigateToAsset(token);
-    await app.account.navigateToToken(token);
-    await app.account.expectLastOperationsVisibility();
+      const app = new Application(page);
+
+      await app.layout.goToPortfolio();
+      await app.portfolio.navigateToAsset(token.tokenName);
+      await app.account.navigateToToken(token);
+      await app.account.expectLastOperationsVisibility();
```

Comment:
Shouldn't we also check that we are on the correct account ?

Reply:
done